src/actions/run/pullRequestAzureRepo.ts

```diff
@@ -109,13 +108,13 @@ export const pullRequestAzureRepoAction = (options: {
         title: title,
       } as GitInterfaces.GitPullRequest;
 
-      const org = ctx.input.organization ?? 'notempty';
-      const token = ctx.input.token ?? integrationConfig.config.token!;
-
       await createADOPullRequest({
         gitPullRequestToCreate: pullRequest,
         server: server,
```

Comment:
Does `server` here need to be `host` instead?

```suggestion
        server: host,
```

`server` can be undefined if it wasn't specified in the template, and in that case it should default to `dev.azure.com` which is what `host` does

Reply:
I think this comment no longer applies with my latest changes

Reply:
I think it's still needed - createADOPullRequest() uses the value passed in here to generate the URL that it actually makes the API call to. It ends up trying to connect to https://undefined/org_name even though it's got the right token.

Reply:
@jackmtpt I think you are right, I was thrown off a bit because, as far as I can tell from the diff, the original implementation passed `server` instead of `host` to `createADOPullRequest`.

I'm passing `host` to `createADOPullRequest` to make sure that whatever value is used to resolve the credentials is also used passed to the helper method.

Reply:
Looks good now, cheers :)